qcengine/programs/entos.py

```diff
@@ -85,7 +107,7 @@ def execute(self,
             infiles.update(extra_infiles)
 
         # Collect all output files and extend with with extra_outfiles
-        outfiles = ["dispatch.out"]
```

Comment:
Do we need to bump the minimum version to 0.6?

Reply:
Yes, I'll make that change.